pkg/network/admitter/netsource.go

```diff
@@ -68,3 +68,16 @@ func validateSingleNetworkSource(field *k8sfield.Path, spec *v1.VirtualMachineIn
 	}
 	return causes
 }
+
+func validateMultusNetworkSource(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec) []metav1.StatusCause {
+	for idx, net := range spec.Networks {
+		if net.Multus != nil && net.Multus.NetworkName == "" {
+			return []metav1.StatusCause{{
+				Type:    metav1.CauseTypeFieldValueRequired,
+				Message: "CNI delegating plugin must have a networkName",
```

Comment:
nit:
CNI delegating plugin sounds too have too much details, shouldnt it just say net-attach-def name is missing? 
```suggestion
				Message: "networkName must be spesifed.",
```